### DISCUSSION TITLE
Port forwarding not working in iOS

### DISCUSSION THREAD

**Discussion by last-Programmer** [utmapp/UTM#5345] [UTM Help (iOS)]
Hi,

I have installed alpine linux arm64 edition in iPad Pro M1 and setup visual studio code server using http://coder.com and the coder server listening at port 3000. I forwarded the port in network setting to hosts 3000 port and when i try to access http://localhost:3000 i am getting error message unreachable. coder.com creates a tunnel and gives public url and when i access it with public url it is working.

Any help would be appreciated

Thank You

**Comment by last-Programmer** [comment]
I even tried to create a sample asp net core application and i cant browse the application from the host through port forwarding. Do we have the option of bridged networking in iOS?

Thanks

**Comment by last-Programmer** [comment]
It seems to be working fine. I have to start the web server with 0.0.0.0:port and not 127.0.0.1:port in the guest.

**Comment by Reidond** [comment]
Hi, how is experience coding like that in linux VM and code-server instance?

**Comment by last-Programmer** [comment]
it is very slow and not a good way to code. i am using now mini pc now. i rdp from ipad  or run vscode remotely and access it from ipad.
